firefish/fluids.py

```diff
@@ -0,0 +1,54 @@
+"""
+This module provides shortcuts to commonly used fluids
+"""
+
+import enum
+
+from firefish.case import (
+    FileName)
+
+class Fluid(enum.Enum):
+    """
+    An enumeration of commonly used fluids
+
+    AIR generates the recommended OpenFoam thermophysicalProperties for air.
```

Comment:
Is there something in Google's style guide about enum-likes? I'd have expected something like

```
Attributes:
    AIR: the recommended OpenFoam thermophysicalProperties for air
    DIMENSIONLESS_AIR: a normalised gas whith gamma=7/5 and...
```

But this is based entirely on what looks surprising. I've no evidence on what is recommended. That being said, listing them as attributes will probably make the docs look nicer.